examples/remote/binary_dependencies/cargo/crates.bzl

```diff
@@ -9,6 +9,144 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")  # bui
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")  # buildifier: disable=load
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")  # buildifier: disable=load
 
+# EXPERIMENTAL -- MAY CHANGE AT ANY TIME: A mapping of package names to a set of normal dependencies for the Rust targets of that package.
+_DEPENDENCIES = {
+    "remote/binary_dependencies": {
+        "ferris-says": "@remote_binary_dependencies__ferris_says__0_2_0//:ferris_says",
+    },
+}
+
+# EXPERIMENTAL -- MAY CHANGE AT ANY TIME: A mapping of package names to a set of proc_macro dependencies for the Rust targets of that package.
+_PROC_MACRO_DEPENDENCIES = {
+    "remote/binary_dependencies": {
+    },
+}
+
+# EXPERIMENTAL -- MAY CHANGE AT ANY TIME: A mapping of package names to a set of normal dev dependencies for the Rust targets of that package.
+_DEV_DEPENDENCIES = {
+    "remote/binary_dependencies": {
+    },
+}
+
+# EXPERIMENTAL -- MAY CHANGE AT ANY TIME: A mapping of package names to a set of proc_macro dev dependencies for the Rust targets of that package.
+_DEV_PROC_MACRO_DEPENDENCIES = {
+    "remote/binary_dependencies": {
+    },
+}
+
+def crates(deps):
+    """EXPERIMENTAL -- MAY CHANGE AT ANY TIME: Finds the fully qualified label of the requested crates for the package where this macro is called.
+
+    WARNING: This macro is part of an expeirmental API and is subject to change.
+
+    Args:
+        deps (list or str): Either a list of dependencies or a string of one which will
```

Comment:
Can this just take a list? I don't think the flexibility here makes this easier to use.

Reply:
Yeah, would changing this also address https://github.com/google/cargo-raze/pull/319#discussion_r548356293 and https://github.com/google/cargo-raze/pull/319#discussion_r548356156 ?

Reply:
No. The weird type thing here is separate from "dependencies" being insufficiently specific.

Reply:
Updated